llvm_patches/16_0_17_0_-X86-getFauxShuffleMask-handle-fp-int-bitcast.patch

```diff
@@ -0,0 +1,39 @@
+From 7aa6ddfea210b6b114ac649c4d3219138e9cc52a Mon Sep 17 00:00:00 2001
```

Comment:
Why underscore after `17_0`? Do we need that for pattern matching?

Reply:
No, it is just an artifact of copy-pasting.